operator/roles/v1.12/kiali-deploy/defaults/main.yml

```diff
@@ -13,8 +13,11 @@ kiali_defaults:
   istio_namespace: ""
   version: "v1.12"
 
-  additional_display_details: []
-
+  additional_display_details:
```

Comment:
This file is in `roles/v1.12` - are you sure you want to edit this file?

You mentioned elsewhere that there is no backporting required for this PR. I think you mean to change the defaults/main.yml in the `roles/default` ansible role location.

Reply:
oops, good catch, I don't know how I came up modifying the wrong one!